backend/wordlist/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.wordlist, name='wordlist'),
+    path('add', views.wordlist_add, name='wordlist_add'),
+    path('remove', views.wordlist_remove, name='wordlist_remove'),
```

Comment:
I think it has to be compliant with our design document.

Reply:
Changed this to meet Restful design. Added a parameter action though, to distinguish add and remove.